src/main/scala/org/apache/spark/sql/internal/oap/OapConf.scala

```
@@ -147,6 +147,13 @@ object OapConf {
       .booleanConf
       .createWithDefault(true)
 
+  val OAP_ENABLE_OPTIMIZATION_STRATEGIES =
+    SQLConfigBuilder("spark.sql.oap.strategies.enabled")
+      .internal()
+      .doc("To indicate if enable/disable oap strategies")
+      .booleanConf
+      .createWithDefault(false)
```

why default false?

---

By default index selection is on, so oapstrategy can not work. Here I think default true value can be misleading.